fixtures/miniflare-node-test/src/index-build.test.js

```diff
@@ -0,0 +1,41 @@
+import assert from "node:assert";
+import { spawnSync } from "node:child_process";
+import test, { after, before, describe } from "node:test";
+import { Miniflare } from "miniflare";
+
+before(() => {
+	spawnSync("npx wrangler build -c wrangler-build.json", {
+		shell: true,
+		stdio: "pipe",
+	});
+});
+
+describe("worker build", () => {
```

Comment:
Very nice PR!

nit: "worker build" and "worker" names in the next file are not very descriptive.

(question: how to use a different worked when interacting with Miniflare? I don't see an option so I guess `MINIFLARE_WORKERD_VERSION` is respected?)

Reply:
`MINIFLARE_WORKERD_VERSION` should be respected, yes

Reply:
Do you think it would be worth adding some docs [on the class](https://github.com/cloudflare/workers-sdk/blob/59c7c8ee177d9345948a416377c6625269d58925/packages/miniflare/src/index.ts#L706)?

Reply:
It's documented in the README: https://github.com/cloudflare/workers-sdk/tree/main/packages/miniflare/README.md#L873